conductor/lib/common.py

```diff
@@ -509,6 +509,8 @@ def get_config_file_paths(self, config_file=None):
             possible_paths = [x for x in os.environ['CONDUCTOR_CONFIG'].split(path_separator) if len(x) > 0]
             if len(possible_paths) > 0:
                 return possible_paths
+        if platform.system() == 'Windows':
+            return [os.path.join(os.environ['APPDATA'], 'Conductor Technologies', 'Conductor', 'config.yml')]
```

Comment:
The client's preferences directory should really live as either a constant or retrieved by a getter function.  Not sure if the installer is responsible/knowledgeable of creating a user prefs directory for Conductor, but obviously the python code should by in sync with it (and ideally tightly bound to it, i.e. DRY).

Reply:
@lawschlosser do you feel strongly enough about this to create a Jira ticket?

Reply:
this is going be addressed by CR-462

Reply:
I hate writing jira tickets.... especially for other people.   yes, I do feel strongly about it ;)